tests/quantization/torchao/test_torchao.py

```diff
@@ -637,3 +658,25 @@ def test_quantization(self):
             gc.collect()
             torch.cuda.empty_cache()
             torch.cuda.synchronize()
+
+    def test_serialization(self):
```

Comment:
I would also add a test with a real model to check memory usage
https://github.com/huggingface/diffusers/blob/6dfaec348780c6153a4cfd03a01972a291d67f82/tests/quantization/gguf/test_gguf.py#L59

Reply:
I've added two tests - one for int4wo (which should use quarter the memory of the original bf16 weights), and one for int8wo (which should use half the memory)